BikeRental/BikeRental.Application/Services/BikeModelService.cs

```diff
@@ -0,0 +1,61 @@
+using BikeRental.Application.Contracts.Dtos;
+using BikeRental.Application.Interfaces;
+using BikeRental.Application.Mappings;
+using BikeRental.Domain.Interfaces;
+
+namespace BikeRental.Application.Services;
+
+/// <summary>
+/// Application-сервис для работы с моделями велосипедов. Инкапсулирует бизнес-логику и доступ к репозиторию.
+/// </summary>
+public sealed class BikeModelService(IBikeModelRepository bikeModelRepository) : IBikeModelService
+{
+    public async Task<IEnumerable<BikeModelDto>> GetAll()
+    {
+        return (await bikeModelRepository.GetAll()).
+            Select(bm => bm.ToDto());
+    }
+
+    public async Task<BikeModelDto?> GetById(int id)
+    {
+        return (await bikeModelRepository.GetById(id))?.ToDto();
+    }
+
+    public async Task<BikeModelDto> Create(BikeModelCreateUpdateDto dto)
+    {
+        var id = await bikeModelRepository.Add(dto.ToEntity());
+        if (id > 0)
+        {
+            var createdEntity = await bikeModelRepository.GetById(id);
+            if (createdEntity != null)
+            {
+                return createdEntity.ToDto();
+            }
+        }
+        throw new InvalidOperationException("Failed to create entity.");
+    }
+
+    public async Task<BikeModelDto> Update(int id, BikeModelCreateUpdateDto dto)
+    {
+        var createdEntity = await bikeModelRepository.GetById(id)
```

Comment:
Если не используете лучше объявлять так:
```suggestion
        _ = await bikeModelRepository.GetById(id)
```